### ISSUE TITLE
There is no Google Store among Apps

### ISSUE THREAD

**Issue by 512yottabytes** [geeks-r-us/anbox-playstore-installer#69]
There is no Google Store among Apps.

**Comment by happyeggchen** [comment]
Yes.Due to the newest version in x86_64 has nothing in it so you have to change the source manually to the https://sourceforge.net/projects/opengapps/files/x86_64/20210701/
then it can be installed

**Comment by happyeggchen** [comment]
modify this
```
OPENGAPPS_RELEASEDATE="$($CURL -s https://api.github.com/repos/opengapps/x86_64/releases/latest | grep tag_name | grep -o "\"[0-9][0-9]*\"" | grep -o "[0-9]*")"
```
to
OPENGAPPS_RELEASEDATE="20210701"

**Comment by happyeggchen** [comment]
However this is not a long-time solution,maybe the package in latest Opengapps release will back,idk.

**Comment by Argus-Kan** [comment]
> modify this
> 
> ```
> OPENGAPPS_RELEASEDATE="$($CURL -s https://api.github.com/repos/opengapps/x86_64/releases/latest | grep tag_name | grep -o "\"[0-9][0-9]*\"" | grep -o "[0-9]*")"
> ```
> 
> to
> 
> ```
> OPENGAPPS_RELEASEDATE="20210701"
> ```

hey I had the same issue so I did what your suggested:
![image](https://user-images.githubusercontent.com/44739457/127415082-ab0625a7-6a37-45d3-a206-8e28ef7f73c4.png)

but now it's giving me different error. 😢
![image](https://user-images.githubusercontent.com/44739457/127415633-d5ba622b-73ab-4cd7-88c3-b12629795168.png)

**Comment by Argus-Kan** [comment]
Never mind, I tried running it again and somehow it ran perfectly fine this time.

**Comment by brudel** [comment]
Looks like it's no more a problem.
